master/internal/api/logs.go

```diff
@@ -98,23 +111,16 @@ func (l *LogStoreProcessor) Receive(ctx *actor.Context) error {
 					return nil
 				}
 			}
-			actors.NotifyAfter(ctx, 500*time.Millisecond, tick{})
 		default:
 			l.req.Limit -= batch.Size()
 			l.req.Offset += batch.Size()
 			switch err := l.process(batch); {
 			case err != nil:
 				return fmt.Errorf("failed while processing batch: %w", err)
-			case !l.req.Follow && l.req.Limit <= 0:
+			case l.req.Limit <= 0:
```

Comment:
blocking: I think this should be reverted or something? Try running `det e create -f ...` -- I'm seeing that exit after reading just a few log lines. (I also once saw it exit with a "404 trial not found" for the first time ever, but I'm not _quite_ as sure that that's related.)

Reply:
yeah sorry, this was a remnant from when i was trying to figure out why streams weren't terminating when they should. ended up needing to add the bit to check if we weren't following and the batch was nil to drop out. i could've sworn i added this at some point but I must've lost it.